staging/src/k8s.io/kubectl/pkg/cmd/version/version.go

```diff
@@ -116,32 +118,32 @@ func (o *Options) Validate(args []string) error {
 // Run executes version command
 func (o *Options) Run() error {
 	var (
-		serverVersion *apimachineryversion.Info
-		serverErr     error
-		versionInfo   Version
+		serverErr   error
+		versionInfo Version
 	)
 
-	clientVersion := version.Get()
```

Comment:
I got rid of the separate serverVersion and clientVersion variables in favour of having all usages rely on the same copy of the data, i.e. the one in `versionInfo`